protocol/rtmp/rtmp.go

```diff
@@ -138,6 +146,7 @@ func (s *Server) handleConn(conn *core.Conn) error {
 		flvWriter := new(flv.FlvDvr)
 		s.handler.HandleWriter(flvWriter.GetWriter(reader.Info()))
 	} else {
+		configure.RoomKeys.GetKey(name) // set new key if this channel not exists
```

Comment:
By connecting to any new channel at rtmp://.../app/chan_xxx,   a new key for this channel will be created and print at console.
This is a temporary hack. Try to create a new interface to manage channels(rooms) and keys.